### PR TITLE
Add MuParser

### PR DESCRIPTION
**Pull request by uilianries**
Specify library name and version:  **muparser/2.3.2**

One more library from Conan Community.

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
Download error

**Comment by uilianries** [comment]
Download error (again)

File: recipes/muparser/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import glob
```
Comment (prince-chrismc) [inline recipes/muparser/all/conanfile.py:1]: dont see it being used

```suggestion
```

File: recipes/muparser/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import os
+from conans import ConanFile, CMake, tools
+
+
+class MuParserConan(ConanFile):
+    name = "muparser"
+    license = "MIT"
+    homepage = "http://beltoforion.de/article.php?a=muparser"
```
Comment (prince-chrismc) [inline recipes/muparser/all/conanfile.py:9]:
```suggestion
    homepage = "https://beltoforion.de/en/muparser/"
```

**Comment by SpaceIm** [comment]
@uilianries `config.yml` is missing

File: recipes/muparser/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import glob
+import os
+from conans import ConanFile, CMake, tools
+
+
+class MuParserConan(ConanFile):
+    name = "muparser"
+    license = "MIT"
```
Comment (SpaceIm) [inline recipes/muparser/all/conanfile.py:8]:
```suggestion
    license = "BSD-2-Clause"
```

File: recipes/muparser/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/muparser/all/conanfile.py]: Done

File: recipes/muparser/all/conanfile.py (hunk 2 above)
Comment (uilianries) [reply recipes/muparser/all/conanfile.py]: Done

File: recipes/muparser/all/conanfile.py (hunk 3 above)
Comment (uilianries) [reply recipes/muparser/all/conanfile.py]: Done

**Comment by uilianries** [comment]
@SpaceIm @prince-chrismc Thank you for your review. All suggestions have been applied.

Let's see if the network error will persist

File: recipes/muparser/all/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "muparser"
+        self.cpp_info.names["cmake_find_package_multi"] = "muparser"
+        self.cpp_info.names["pkg_config"] = "muparser"
+        self.cpp_info.libs = ["muparser"]
+        if not self.options.shared:
+            self.cpp_info.defines = ["MUPARSER_STATIC=1"]
```
Comment (SpaceIm) [inline recipes/muparser/all/conanfile.py:79]:
```suggestion
            self.cpp_info.defines = ["MUPARSER_STATIC=1"]
        if not self.options.shared and tools.stdcpp_library(self):
            self.cpp_info.system_libs.append(tools.stdcpp_library(self))
```
for C API

**Review by prince-chrismc — APPROVED** [review]

**Comment by uilianries** [comment]
Jenkins is a bit temperamental.

**Review by SpaceIm — APPROVED** [review]

**Pull request merged**